src/Raven.Client/Documents/Commands/DeleteRevisionsManuallyCommand.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations.Revisions;
+using Raven.Client.Extensions;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Json.Serialization;
+using Raven.Client.Util;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Commands
+{
+    internal class DeleteRevisionsManuallyCommand : RavenCommand<DeleteRevisionsManuallyOperation.Result>
```

Comment:
I removed from everything, except DeleteRevisionsManuallyOperation and DeleteRevisionsManuallyCommand (on the client), because there are already classes with the names ""DeleteRevisionsOperation" and "DeleteRevisionsCommand", and I wanted to avoid confusions